### DISCUSSION TITLE
GitHub staff took away my Sponsors page

### DISCUSSION THREAD

**Discussion by girish17** [community/community#52882] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

GitHub recently killed my [sponsorship page](https://github.com/sponsors/girish17) and sponsors account without any intimation or explanation. This is unfortunate for a user like me who had been using GitHub for about eight years now. The sponsorship page was active for about six months and was being supported by supporters without any hassles. Now, all of a sudden, GitHub staff decide to pull down curtains on my sponsorship account, without any reason. At least, I should be told what went wrong and how that could've been rectified. This is just reckless and uncalled for. They even went on to flag and deactivate one of my sponsor's account. 

I feel that the open source developers on GitHub are treated like second-class citizens by GitHub staff and management. The policies are not developer friendly and give more power to GitHub staff over the users of the platform. They can just deactivate anybody's account without any explanation for violation of their draconian policies - utterly whimsical !

@github

**Comment by girish17** [comment]
The payout funds are withheld by GitHub. They have not refunded it to the sponsor or initiated a payout yet. This is just unfair.
@github

**Comment by DHayes10** [comment]
Hi @girish17, 

Thank you for reaching out. I am sorry to hear you are having issues with your Sponsors account. Sometimes our system highlights accounts that need to be reviewed. Please allow us some time to research the latest transactions. Once our investigation is complete, we will reach out for further instructions. 

Thank you, 
GitHub Security Team

**Comment by girish17** [comment]
Dear GitHub Security Team,
Is there any update on this? May I know by when can I expect a resolution?
@DHayes10 @github
